projects/pap/src/app/features/push-notification/push-notification-page.component.ts

```diff
@@ -24,7 +24,10 @@ export class PushNotificationsPageComponent implements OnInit {
   ngOnInit(): void {
     this._store.dispatch(removeAllDeliveredNotifications());
     this.pushNotifications$
-      .pipe(take(1))
+      .pipe(
+        filter(notifications => !!notifications && notifications.length > 0),
+        take(1),
+      )
       .subscribe((notifications: PushNotification[] | undefined) => {
         if (notifications && notifications.length > 0) {
           this.firstNotificationId$.next(notifications[0].id);
```

Comment:
firstNotification$ non serve che sia behaviour subject si puo eliminare la subscribe e fare una cosa del genere firstNotificationId = this.pushNotifications$
      .pipe(
        filter(notifications => !!notifications && notifications.length > 0),
        take(1),
map(n=> n[0].id)
      )